dags/atd_knack_detectors.py

```diff
@@ -94,7 +91,7 @@
         image=docker_image,
         docker_conn_id="docker_default",
         auto_remove="force",
-        command=f"./atd-knack-services/services/records_to_socrata.py -a {app_name} -c {container} {date_filter_arg}",
+        command=f"./atd-knack-services/services/records_to_socrata.py -a {app_name} -c {container}",
```

Comment:
so the date_filter_arg command causes it to not refresh daily? how often was it refreshing before?

Reply:
if a date_filter_arg is supplied to the scripts in knack services, it looks for records updated after the timestamp -- without a date_filter_arg the script does a full replace every time

previously it was doing a full replace on the first of the month, but records that were deleted (in knack) in the interim weren't being removed from the open data set. so there was a mismatch. now we are just doing a full replace every day.